net/netatalk/Makefile

```diff
@@ -8,81 +8,107 @@
 
 define Package/netatalk
-  SECTION:=net
-  CATEGORY:=Network
-  SUBMENU:=Filesystem
-  DEPENDS:=+libattr +libdb47 +libgcrypt +libopenssl +libevent2
-  TITLE:=netatalk
-  URL:=http://netatalk.sourceforge.net
-  MAINTAINER:=An Pa <apccv@outlook.com>
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Filesystem
+	TITLE:=Netatalk - Open Source file server that implements the Apple Filing Protocol (AFP)
+	URL:=http://netatalk.sourceforge.net
+	#DEPENDS:=+libevent2 +libdb47 +libgcrypt +dbus
+	DEPENDS:=+libevent2 +libdb47 +libgcrypt
 endef
 
 define Package/netatalk/decription
-  Netatalk is a freely-available Open Source AFP fileserver.
-  It also provides a kernel level implementation of the AppleTalk
-  Protocol Suite.
+	Netatalk is an Open Source Apple Filing Protocol (AFP) fileserver.
+	Implements a service that allows Macintosh file sharing
+	and Time Machine backups. Modern MacOS prefers Samba network shares
+	but AFP has shown superior throughput. High Sierra and older MacOS
+	only support AFP for Time Machine backups.
+	Does not support old AppleTalk protocols to talk to non OSX Macs.
 endef
 
-TARGET_CFLAGS += -std=gnu99
-
-CONFIGURE_ARGS += \
-	--disable-afs \
-	--disable-hfs \
-	--enable-debugging \
-	--disable-shell-check \
-	--disable-timelord \
-	--disable-a2boot \
-	--disable-cups \
-	--disable-tcp-wrappers \
-	--with-cnid-default-backend=dbd \
-	--with-bdb="$(STAGING_DIR)/usr/" \
-	--with-libevent=no \
-	--with-libgcrypt-dir="$(STAGING_DIR)/usr" \
-	--with-ssl-dir="$(STAGING_DIR)/usr" \
-	--with-uams-path="/usr/lib/uams" \
-	--without-acls \
-	--without-kerberos \
-	--without-mysql \
-	--with-mysql-config=false \
-	--without-pam \
-	--disable-admin-group \
-	--disable-srvloc \
-	--disable-zeroconf \
-	$(if $(CONFIG_SHADOW_PASSWORDS),--with-shadow,--without-shadow) \
-	--without-dtrace \
-	--without-ldap
+MESON_ARGS += \
+	-Dwith-appletalk=false \
+	-Dwith-zeroconf=false \
+	-Dwith-cups=false \
+	-Dwith-quota=false \
+	-Dwith-afpstats=true \
+	-Dwith-acls=false \
+	-Dwith-ldap=false \
+	-Dwith-cnid-default-backend=dbd \
+	-Dwith-cnid-dbd-backend=true \
+	-Dwith-cnid-mysql-backend=false \
+	-Dwith-pam=false \
+	-Dwith-tests=false \
+	-Dwith-testsuite=false \
+	-Dwith-bdb-path="$(STAGING_DIR)/usr/" \
+	-Dwith-unicode-data-path="$(PKG_BUILD_DIR)/" \
+	-Dwith-libgcrypt-path="$(STAGING_DIR)/usr/" \
+	-Dwith-kerberos=false \
+	-Dwith-krbV-uam=false \
+	-Dwith-libiconv=false \
+	-Dwith-spotlight=true \
+	$(if $(CONFIG_SHADOW_PASSWORDS),-Dwith-shadow=true,-Dwith-shadow=false) \
+	-Dwith-dtrace=false \
+	-Dwith-tcp-wrappers=false \
+	-Dwith-debug=false \
+	-Dwith-shell-check=false \
+	-Dwith-readmes=false \
+	-Dwith-manual=none \
+	-Dwith-init-style=none \
+
+	#-Dwith-libiconv-path="$(STAGING_DIR)/usr/" \
+	#-Dwith-uams-path="/usr/lib/uams" \
+	#-Dwith-ssl-dir="$(STAGING_DIR)/usr" \
+	#-Ddisable-admin-group=true \
+	#-Ddisable-srvloc=true \
 
 define Package/netatalk/conffiles
-/etc/afp.conf
-/etc/extmap.conf
-/etc/netatalk/
+	/etc/afp.conf
```

Comment:
Why added indentation here? Thats not correct.

Reply:
I formatted everything thinking all `define Package` had tab-nested comments.
I see now this one in particular is different, no indent.
I see half of the sections have only two-space indent.
Will update overall.

Reply:
Tabs - inherited from make, where commands needs to be indented with tabs.
Conffiles should not have any indentation, otherwise it does not work. You can try it yourself on the router or also by checking the .ipk with mc utility.